CMakeLists.txt

```diff
@@ -2899,6 +2901,11 @@ if(WAVPACK)
   target_link_libraries(mixxx-lib PRIVATE WavPack::wavpack)
 endif()
 
+# Tensorflow for spleeter inference
+target_include_directories(mixxx-lib PUBLIC /usr/include/tensorflow)
```

Comment:
This will fail on Windows etc.

Any ideas to solve this?

Reply:
You need to add the tensorflow package (https://vcpkg.info/port/tensorflow or https://vcpkg.info/port/tensorflow-cc) to https://github.com/mixxxdj/vcpkg first.

Reply:
And you also need to use tensorflow via the `find_package` methanism, not link absolute paths directly. The current instruction will also fail on Linux if tensorflow is installed to `/opt` or `/usr/local` instead of `/usr`.

Reply:
@Holzhaus I was aware that I was making mistakes with CMake and that you'd come to the rescue. Thank you. This means I have to add a FindTensorflow.cmake to cmake/modules, right? Also I should probably add a STEM definition to set ON/OFF together with a __STEM__compiler definition to guard stem code with #ifdef's. Does that sound plausible? Lastly, going off on a weird tangent, I came across your personal uni website through Google a few months ago trying to prevent my wireless network from being flooded by zita-j2n UDP packets and your tips helped A LOT. Thanks!